### ISSUE TITLE
SSL

### ISSUE THREAD

**Issue by uniphil** [Queens-Hacks/qcumber-api#4]
they're fairly cheap at namecheap. low low priority for now, especially since at this point we don't really have privacy to think about...

**Comment by mystor** [comment]
Yeah, I support the idea of having a SSL key, I don't care where we get it, and I don't think we should be worrying about it right now.
